### DISCUSSION TITLE
播放音乐时，lrc歌词卡住不动

### DISCUSSION THREAD

**Discussion by shanyan-wcx** [hooke007/mpv_PlayKit#343] [mpv 通用]
播放音乐时，lrc歌词一直显示同一句不变，只有在切换窗口模式与全屏模式时歌词才会变。

https://github.com/hooke007/MPV_lazy/assets/58252651/ad0d9546-3faf-4c6c-9346-1a59426fbbb0

**Comment by hooke007** [comment]
无样本复现

**Reply by shanyan-wcx** [reply]
我排查了一下，当使用vo = gpu-next时存在这个问题，vo = gpu就没问题。

**Reply by shanyan-wcx** [reply]
解决了，vo = gpu-next和blend-subtitles = video同时开的时候就会出问题，把后者关了就好了。

**Reply by hooke007** [reply]
后续如有修复，关注 https://github.com/mpv-player/mpv/issues/11335
